### PR TITLE
Build all GC retail ROMs in Jenkins

### PR DESCRIPTION
**Pull request by cadmic**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -24,52 +24,68 @@ pipeline {
                 sh 'python3 tools/check_format.py --verbose --compare-to origin/main'
             }
         }
-        stage('Setup gc-eu-mq-dbg') {
+        stage('Build gc-eu-mq-dbg') {
             steps {
                 sh 'cp /usr/local/etc/roms/oot-gc-eu-mq-dbg.z64 baseroms/gc-eu-mq-dbg/baserom.z64'
                 sh 'make -j setup'
+                sh 'make -j RUN_CC_CHECK=0'
+                sh 'make clean assetclean'
```
Comment (Dragorn421) [inline Jenkinsfile:32]:
```suggestion
                sh 'make -j setup VERSION=gc-eu-mq-dbg'
                sh 'make -j RUN_CC_CHECK=0 VERSION=gc-eu-mq-dbg'
                sh 'make clean assetclean VERSION=gc-eu-mq-dbg'
```
reasoning:
- consistency with the rest
- we may remove having a default version eventually?

**Review by Dragorn421 — COMMENTED** [review]
I think we should have the versions built in an order that "maximizes compiler flags coverage", in a "fail fast" approach
For example gc-jp and gc-jp-mq are very close, so if gc-jp and a mq version have already been successfully built we can build gc-jp-mq last, after versions less likely to match than gc-jp-mq
Coming up with this order sounds like an artistic approximation job. Probably should also start with the versions more likely to break (thinking of you, bss), so not (gc-eu-mq-)debug

maybe:
```
gc-jp // REGION_JP OOT_NTSC
gc-eu-mq // REGION_EU OOT_PAL OOT_MQ
gc-eu-mq-dbg // OOT_DEBUG
gc-us // REGION_US
gc-jp-ce
gc-eu
gc-jp-mq
gc-us-mq
```

-------

Other comments:
- instead of `cp`ing the roms we could symlink `ln -s` them, that'd save half a gigabyte (eventually one GB)
- build versions in parallel (apparently Ethan preempted this :sunglasses:)

**Comment by cadmic** [comment]
Alright, marked as ready for review. I think I'm not going to mess with parallelism for now since we only have 2 build machines now anyway.

Some stats: building each ROM takes about 1m30s - 1m40s, and the total build time is 11m

**Review by Dragorn421 — APPROVED** [review]
good bye progress.py

**Review by fig02 — APPROVED** [review]

**Pull request merged**